src/basic/virt.h

```diff
@@ -5,7 +5,7 @@
 
 #include "macro.h"
 
-enum {
+typedef enum Virtualization {
```

Comment:
btw, the reason this was an anonymous so far was that the main function returning this (`detect_virtualization()`) was supposed to also return negative error codes, and hence it made more sense using `int` for it, and don't bother with a name of the type. Not sure myself though whether that reasoning is strictly true though

Reply:
OK. This change is dropped.